config.toml

```diff
@@ -117,6 +117,11 @@ name = "JS Core API"
 url = "https://github.com/ipfs/interface-ipfs-core/tree/master/SPEC"
 weight = 30
 
+[[menu.reference]]
+parent = "go_implementation"
+name = "Recent Releases"
+url = "https://docs-beta.ipfs.io/recent-releases/"
```

Comment:
This URL doesn't exist yet, but will when the `ipfs/ipfs-docs-v2:go-ipfs-0.5` branch is merged into `master`.